lua/lspconfig/server_configurations/sourcery.lua

```diff
@@ -9,6 +9,25 @@ local root_files = {
   'pyrightconfig.json',
 }
 
+local get_token_from_auth_file = function()
+  local is_windows = vim.fn.has 'win32' == 1
+  local path_sep = is_windows and '\\' or '/'
+
+  local config_home = is_windows and vim.fn.getenv 'APPDATA'
```

Comment:
maybe we can use `stdpath` for it ?

Reply:
is the stdpath always gonna be APPDATA or XDG_CONFIG_HOME? token file will in those folders

Reply:
I know it will check `XDG` . I don't know how it works on windows.

Reply:
It looks like stdpath is resolved to APPDATALOCAL in win: https://github.com/neovim/neovim/blob/4b02291f81b76eda029d4eae1c1d59c1a946a032/src/nvim/os/stdpaths.c#L27, so I wouldn't use it here

Reply:
but I can can still fix unix ones

Reply:
actually, `vim.fn.stdpath` returns `XDG_CONFIG_HOME/nvim`, so I don't think it's that usable? Why not just use `vim.fn.getenv(XDG_CONFIG_HOME)` which resolves directly to correct config root folder?

Reply:
checking sourcery docs again, it looks like in *nix config files are always in `~/.config/sourcery` so I'm going to use that instead of resolving from `XDG_CONFIG_HOME`